libs/evm-protocols/src/common-protocol/contractHelpers/launchpad.ts

```diff
@@ -339,3 +339,20 @@ export async function transferLaunchpadLiquidityToUniswap({
     maxFeePerGas,
   );
 }
+
+// Function to get the transaction sender
+export async function getTransactionSender(
```

Comment:
We have an existing `getTransaction` method in `libs/evm-protocols` that we should. The `from` field should be extracted from that instead.